hyperapp.d.ts

```diff
@@ -0,0 +1,45 @@
+export as namespace hyperapp
+
+// VNode part
+
+export interface VNode<Data> {
+  tag: string
+  data?: Data
+  children: VElement<{}>[]
+}
+
+export type VElement<Data> = VNode<Data> | string
+
+export interface Component<Data> {
+  (data?: Data, ...children: VElement<{}>[]): VNode<Data>
+}
+
+export function h<Data>(
+  tag: Component<Data> | string,
+  data?: Data,
+  children?: VElement<{}>[]
+): VNode<Data>
+export function h<Data>(
+  tag: Component<Data> | string,
+  data?: Data,
+  ...children: VElement<{}>[]
+): VNode<Data>
+
```

Comment:
I am looking for a way to express children could be an array ov VElements or the rest of args could be too.

In Snabbdom-pragma, I am using the following
```tsx
type Children = VNode<{}>[] | VNode<{}> | string | number
type CircularChildren = Children | Children[]

export function h<Data>(
   tag: Component<Data> | string,
   data?: Data,
   ...children: CircularChildren 
 ): VNode<Data>
```

`CircularChildren` was about a quick fix regarding : 

```jsx
const arr = [
  [<div>Hello1.1</div>, <div>Hello1.2</div>],
  [<div>Hello2.1</div>, <div>Hello2.2</div>]
]
const vnode = <div> { arr } </div>
```